docs/operator-manual/webhook.md

```diff
@@ -36,12 +36,14 @@ In the `argocd-secret` kubernetes secret, configure one of the following keys wi
 provider's webhook secret configured in step 1.
 
 | Provider        | K8s Secret Key                   |
-|-----------------| ---------------------------------|
+|-----------------|----------------------------------|
 | GitHub          | `webhook.github.secret`          |
 | GitLab          | `webhook.gitlab.secret`          |
 | BitBucket       | `webhook.bitbucket.uuid`         |
 | BitBucketServer | `webhook.bitbucketserver.secret` |
 | Gogs            | `webhook.gogs.secret`            |
+| Azure DevOps    | `webhook.azuredevops.username`   |
+|                 | `webhook.azuredevops.password`   |
```

Comment:
as a DevOps user I'd like a bit of clarity on what this password is - is it a PAT or something else?

Reply:
Ah nevermind, I presume when adding the webhook you generate a password and this is to validate it on the argo end
![image](https://github.com/argoproj/argo-cd/assets/2775804/07ac0ed8-387b-484b-add0-d2d9714dfd87)

Reply:
This screenshot might be a good thing to add to docs. :-)

Reply:
agree :) created separate sections for Github and Azure DevOps and used this screenshot in Azure DevOps section.